packages/network/src/handlers/solana.ts

```diff
@@ -84,5 +84,9 @@ export const signAndSendTransactionAbstractionFee = async (
 
 	const data = await res.json();
 
+	if (data.status === 'error') {
+		throw new Error(data.message);
+	}
+
```

Comment:
should check the returned status code inside the response instead of checking body (I think it can be changed).

Reply:
you might want to check `res.ok`